homeassistant/components/permobil/strings.json

```diff
@@ -27,7 +27,8 @@
       "region_fetch_error": "Error fetching regions",
       "code_request_error": "Error requesting application code",
       "invalid_email": "Invalid email",
-      "invalid_code": "The code you gave is incorrect"
+      "invalid_code": "The code you gave is incorrect",
+      "unsigned_eula": "Please sign the EULA in the MyPermobil app and start over"
```

Comment:
With start over you mean the user MUST start the whole flow again after signing?

Reply:
Turns out the email code will be reusable so they do not need to start over from the beginning. 
(and if they still choose to close and restart the config flow it will still work)